compiler/src/iree/compiler/Codegen/Interfaces/PartitionableLoopsInterface.cpp

```diff
@@ -208,7 +208,7 @@ void registerPartitionableLoopsInterfaceModels(DialectRegistry &registry) {
                             IREE::LinalgExt::IREELinalgExtDialect *dialect) {
     IREE::LinalgExt::FftOp::attachInterface<FftOpPartitionableLoops>(*ctx);
     IREE::LinalgExt::PackOp::attachInterface<
-        NoPartitionableLoops<IREE::LinalgExt::PackOp>>(*ctx);
+        OuterParallelAsPartitionableLoops<IREE::LinalgExt::PackOp>>(*ctx);
```

Comment:
The loops of PackOp are all parallel, should this be `AllParallelAsPartitionableLoops`?

(maybe no difference in this case, but it looks more accurate to me.)